.github/workflows/pull-request.yml

```
@@ -0,0 +1,272 @@
+        run: |
+          # Retrieve the complete branch name directly from the GitHub context
+          FULL_BRANCH_NAME=${{ github.base_ref }}
+          # Validate branch name
+          if [[ ! $FULL_BRANCH_NAME =~ ^[a-zA-Z0-9_/-]+$ ]]; then
+            echo "Invalid branch name"
+            exit 1
+          fi
+          echo "FULL_Branch_NAME: $FULL_BRANCH_NAME"
+          
+          # Clone the specified repository using the extracted branch name
+          git clone --branch $FULL_BRANCH_NAME https://github.com/PalisadoesFoundation/talawa-api && ls -a 
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)